bootstrap/app.php

```diff
@@ -79,7 +80,7 @@
 */
 
 $app->register(\Adgangsplatformen\Support\Illuminate\AdgangsplatformenServiceProvider::class);
-$app->register(Spatie\Cors\CorsServiceProvider::class);
+$app->register(CorsServiceProvider::class);
```

Comment:
Is this something that they forgot to mention in the upgrade guide or did we miss something?

Reply:
No it's from the Laravel upgrade guide: https://laravel.com/docs/7.x/upgrade#cors-support